scripts/testnet/include/config.sh

```diff
@@ -106,12 +107,30 @@ updateElasticsearchIndices() {
   popd
 }
 
+updateConfigFile() {
```

Comment:
You also need to call this function for `sovereignConfig.toml`, but **ONLY** for `OutgoingSubscribedEvents`.SubscribedEvents field in this case.

A good code would be to change there everything except what's found under `NotifierConfig` toml table

Reply:
`sovereignConfig.toml` is updated by deploy scripts, I don't need to use this... maybe in the future if would be necessary

Reply:
LE: I need to use this on `sovereignConfig.toml` in case we just start sovereign with dummy addresses. Those addresess are overwritten by the deploy scripts later anyway.